treeherder/perf/management/commands/synthesize_backfill_report.py

```diff
@@ -67,3 +68,16 @@ def _validate_args(self, frameworks: List[str], repositories: List[str]):
         if repositories:
             if not set(repositories).issubset(set(Command.repos_to_retrigger_on)):
                 raise ArgumentError('Unknown repository provided.')
+
+    def _queue_is_too_loaded(self, provisioner_id: str, worker_type: str, acceptable_limit=1000) -> bool:
+        '''
+        Helper method for PerfSheriffBot to check load on processing queue.
+        Usage example: _queue_is_too_loaded('gecko-3', 'b-linux')
+        :return: True/False
+        '''
+        tc = TaskclusterConfig('https://community-tc.services.mozilla.com')
```

Comment:
We need to access the production Taskcluster. I believe that's `https://firefox-ci-tc.services.mozilla.com`.

Reply:
Done